app/services/action_service.rb

```diff
@@ -19,6 +19,10 @@ def change_status(status)
     @conversation.update!(status: status[0])
   end
 
+  def change_priority(priority)
+    @conversation.update!(priority: (priority[0] == 'nil' ? nil : priority[0]))
```

Comment:
The `priority[0]` values are "low", "medium", "high", "urgent", or "nil". If it is "nil", we must set it to `nil`, Otherwise, we need to set the priority.

Reply:
Send `null` instead of `nil` from the frontend. It will convert it to `nil` automatically.

Reply:
We could do that. But we need to tweak some changes in the frontend for automation payload, which may also require changes in other automation rules.

**Reference**
https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/helper/actionQueryGenerator.js#L21
Here we always expect some values in id; if we set it to null, the automation will save it in a different format.